modules/admin-quota-overview.adoc

```diff
@@ -0,0 +1,309 @@
+`*requests.ephemeral-storage*` are the same value and can be used interchangeably. This resource is available only if you enabled the ephemeral
+storage technology preview. This feature is disabled by default.
+
+|`*limits.cpu*`
+|The sum of CPU limits across all pods in a non-terminal state cannot exceed this value.
+
+|`*limits.memory*`
+|The sum of memory limits across all pods in a non-terminal state cannot exceed this value.
+
+|`*limits.ephemeral-storage*`
+|The sum of ephemeral storage limits across all pods in a non-terminal state cannot exceed this value. This resource is available only if you enabled the
+ephemeral storage technology preview. This feature is disabled by default.
+|===
+
+
+.Storage Resources Managed by Quota
+[cols="3a,8a",options="header"]
+|===
+
+|Resource Name |Description
+
+|`requests.storage`
+|The sum of storage requests across all persistent volume claims in any state cannot exceed this value.
+
+|`persistentvolumeclaims`
+|The total number of persistent volume claims that can exist in the project.
+
+|`<storage-class-name>.storageclass.storage.k8s.io/requests.storage`
+|The sum of storage requests across all persistent volume claims in any state that have a matching storage class, cannot exceed this value.
+
+|`<storage-class-name>.storageclass.storage.k8s.io/persistentvolumeclaims`
+|The total number of persistent volume claims with a matching storage class that can exist in the project.
+|===
+
+
+.Object Counts Managed by Quota
+[cols="3a,8a",options="header"]
+|===
+
+|Resource Name |Description
+
+|`pods`
+|The total number of pods in a non-terminal state that can exist in the project.
+
+|`replicationcontrollers`
+|The total number of replication controllers that can exist in the project.
+
+|`resourcequotas`
+|The total number of resource quotas that can exist in the project.
+
+|`services`
+|The total number of services that can exist in the project.
+
+|`secrets`
+|The total number of secrets that can exist in the project.
+
+|`configmaps`
+|The total number of `ConfigMap` objects that can exist in the project.
+
+|`persistentvolumeclaims`
+|The total number of persistent volume claims that can exist in the project.
+
+|`openshift.io/imagestreams`
+|The total number of image streams that can exist in the project.
+
+|===
+end::admin_quota_resources_managed[]
+
+You can configure an object count quota for these standard namespaced resource types using the `count/<resource>.<group>` syntax.
+
+[source,terminal]
+----
+$ oc create quota <name> --hard=count/<resource>.<group>=<quota> <1>
+----
+
+<1> `<resource>` is the name of the resource, and `<group>` is the API group, if applicable.
+Use the `kubectl api-resources` command for a list of resources and their associated API groups.
+
+== Setting Resource Quota for Extended Resources
+
+Overcommitment of resources is not allowed for extended resources, so you must specify `requests` and `limits` for the same extended resource in a quota.
```

Comment:
I probably don't have an idea what overcommitment means here. Also I don't think `overcommitment` is a valid English word. Can we rewrite it to something more specific.

```suggestion
You must specify `requests` and `limits` for the same extended resource in a quota.
```